arangod/VocBase/LogicalCollection.cpp

```diff
@@ -616,7 +627,6 @@ arangodb::Result LogicalCollection::appendVelocyPack(
     // with 'forPersistence' added by LogicalDataSource::toVelocyPack
     // FIXME TODO is this needed in !forPersistence???
     result.add(StaticStrings::DataSourceDeleted, VPackValue(deleted()));
-    result.add(StaticStrings::DataSourceGuid, VPackValue(guid()));
```

Comment:
Why did you remove that ?

Reply:
because previously DataSourceGuid was output by LogicalDataSource if forPersistence and by LogicalCollection if !forPersistence, now it's always output by LogicalDataSource